components/lsp-form/LSPForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, useState, useEffect } from "react";
+import Tabs from "../tabs";
+import {
+  Wrapper,
+  SettleButton,
+  SettleWrapper,
+  SettleTitle,
+  TimeRemaining,
+  SettleText,
+} from "./LSPForm.styled";
+
+import MintForm from "./MintForm";
+import RedeemForm from "./RedeemForm";
+import { calculateTimeRemaining } from "./helpers";
+
+const LSPForm: FC = () => {
+  const [showSettle, setShowSettle] = useState(false);
+
+  // Stub time remaining.
+  const [timeRemaining, setTimeRemaining] = useState("00:00");
+  useEffect(() => {
+    setTimeRemaining(calculateTimeRemaining());
```

Comment:
we will eventually have to set an end date, if u want to do that in a future pr make sure theres an issue for it

Reply:
Yeah I'm not 100% sure where this is coming from and until we get more details I'd say flesh it out later.